lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -1,6 +1,6 @@
 module SmartAnswer::Calculators
```

Comment:
I'll edit the commit note, though referring to a country as "flattened" doesn't read well either.